recipes/tokyocabinet/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "tokyocabinet" %}
+{% set version = "1.4.48" %}
+{% set sha256 = "a003f47c39a91e22d76bc4fe68b9b3de0f38851b160bbb1ca07a4f6441de1f90" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://fallabs.com/tokyocabinet/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+  has_prefix_files:
+    - lib/pkgconfig/tokyocabinet.pc
+
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - zlib
+    - bzip2
```

Comment:
Please pin these to [match]( https://github.com/conda-forge/conda-forge.github.io/blob/f5f6e31d503bc099f1cb9649608187305cd438a6/scripts/pin_the_slow_way.py#L29-L54 ).